developers/weaviate/current/client-libraries/python.md

```diff
@@ -158,6 +173,21 @@ Batching is a way of importing/creating `objects` and `references` in bulk using
 2. ***Dynamic-batching***
 3. ***Manual-batching***
 
+## New: Multi-threading batch import (weaviate-client>=3.9.0)
+Python client version `3.9.0` introduces Multi-threading Batch import which works with both `Auto-batching` and `Dynamic-batching`. You can set the number of workers (threads) can be configured using the `.configure(...)` (same as `.__call__(...)`) by setting the argument `num_workers`. See also *Batch-configuration* below.
```

Comment:
I would add that this is disabled by default

Reply:
Agree